src/ambassadors/core.ts

```diff
@@ -385,6 +396,12 @@ const ambassadors = {
       "He is an ambassador for the exploitation of wildlife in the pet trade and the fur trade.",
     clips: [
       { id: "670RYU2Xw98", caption: "Fenn Digs Up A Friend In The Enclosure" },
+      { id: "awX8PnOfhQA", caption: "Fenn Finds A Well Hidden Treasure" },
+      { id: "ScccoAPYPB0", caption: "Fenn The Fox ASMR" },
+      {
+        id: "9HbrUaQzhXI",
+        caption: "The Foxes Reunite And Fenn Couldn't Be More Excited About It",
```

Comment:
```suggestion
        caption: "The Foxes Are Excited To Be Reunited",
```
Maybe we can use a shorter caption here, as the original title is rather long?